scripts/check-breaking-changes.sh

```diff
@@ -48,8 +48,8 @@ while IFS=" " read -r -a package; do
 done <<< "$PACKAGES"
 
 # "Export" the message to an environment variable that can be used across Github Actions steps
-echo "::set-output name=is_breaking::$EXIT_CODE"
-echo "::set-output name=message::$GITHUB_MESSAGE"
+echo "is_breaking=$EXIT_CODE" >> $GITHUB_OUTPUT
```

Comment:
did not test this, so it would be good to figure out first how to test this change

Reply:
Also shellcheck seems to fail, not sure if I need to add quotes around `$GITHUB_OUTPUT`

https://drone.grafana.net/grafana/grafana/88175/4/3

Reply:
I can test it tomorrow in the [workflow-test](https://github.com/grafana/workflow-test) repo?

Reply:
that would be great, as I am not sure how to do it myself :)

Reply:
Tested it out and looks like it is working: https://github.com/grafana/github-actions-testrepo/actions/runs/3436069084/jobs/5729166822#step:8:8